simulator-starter/pom.xml

```diff
@@ -44,14 +44,14 @@
       <artifactId>h2</artifactId>
     </dependency>
 
-    <dependency>
-      <groupId>javax.transaction</groupId>
-      <artifactId>javax.transaction-api</artifactId>
-    </dependency>
     <dependency>
       <groupId>javax.interceptor</groupId>
       <artifactId>javax.interceptor-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.transaction</groupId>
+      <artifactId>javax.transaction-api</artifactId>
+    </dependency>
```

Comment:
Both dependencies here should also point to the Jakarta artefacts:
https://mvnrepository.com/artifact/jakarta.interceptor/jakarta.interceptor-api/1.2.4
https://mvnrepository.com/artifact/jakarta.transaction/jakarta.transaction-api/1.3.2

Reply:
I agree on all the jakarta-updates 👍

Reply:
Done in 6bd41c1deb2a348d4f6dcee2f6e1f48c0376b6d3.